.env.example

```
@@ -37,7 +37,9 @@ TINYAUTH_USERSFILE=""
 # Enable secure cookies (requires HTTPS)
 TINYAUTH_SECURECOOKIE="true"
 # Session expiry in seconds (7200 = 2 hours)
-TINYAUTH_SESSIONEXPIRY="7200"
+TINYAUTH_AUTH_SESSIONEXPIRY="7200"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)